tests/system/NVDA Core/variables.py

```diff
@@ -0,0 +1,14 @@
+WELCOME_DIALOG_TEXT = (

```

Comment:
This might break if a user runs the system tests with a system language other than English, in which the user default language differs.

Reply:
Yes, it's likely that there will be lots of ways that the tests will not pass on various different systems. To start with, we will be using AppVeyor as the target. Over time, if people are trying to run the tests locally, then the tests will become more tolerant of variations in the underlying system.